de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/LocalEditorHandler.java

```diff
@@ -156,7 +157,7 @@ private Editor openEditor(@NotNull VirtualFile virtualFile,
 
             return null;
 
-        }else if (!manager.getSession().isShared(path.getResource())) {
+        }else if (!SessionUtils.isShared(path)) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "SessionUtils"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=293&issues=AWcxWGhf0POp3Yiz0EfP&open=AWcxWGhf0POp3Yiz0EfP)

[ISSUE_KEY]: AWcxWGhf0POp3Yiz0EfP